utils/scarb-test-support/tests/examples.rs

```diff
@@ -0,0 +1,38 @@
+use scarb_test_support::command::Scarb;
+
+#[test]
+fn test_help() {
```

Comment:
what's the reason behind adding this?

Reply:
It's for cargo test to create the binary "scarb-test-support".
It helps cargo_bin("scarb-test-support").

Reply:
Can you please:
1. Add an explanatory comment to this file.
2. Rename the test `binary_dependencies_hack` and leave its body empty. I do not want to test help output, especially in such exotic place.
3. Rename test file to something more meaningful. Even simple `hack.rs` is better than `examples.rs` IMO

Reply:
And also, please create an issue in our tracker to fix this hack with https://github.com/rust-lang/cargo/issues/9096 once it's released in stable Rust.

Reply:
Done!